### ISSUE TITLE
Rename this module

### ISSUE THREAD

**Issue by fabaff** [antonagestam/nmap#2]
I strongly suggest that you rename your module for the following reasons:

- there is a well-known tool out there called `nmap`. your choice looks very much that you want to piggyback on their name and search engine results
- your module will never make it into the repositories of the distributions because `py-nmap` or `python-nmap` is already taken by the binding of the tool mentioned above

**Comment by antonagestam** [comment]
Thank you for your feedback, I will consider that. However, this is just a 10-line utility library made entirely for my own convenience and I'm more than fine with nobody other than myself actually using it.

I don't think search engines actually work in a way where you can piggyback like you describe, in that case I would have totally named this package facebook-twitter-google-awesome.

Though if you have name suggestions those are welcome.

Cheers

**Comment by streanger** [comment]
@antonagestam
Its not about search engines and its not about your _I'm more than fine with nobody other than myself actually using it_ feeling. My experience over the years is that **everybody** install pypi packages by their import names. So when It comes to install nmap package they will type `pip install nmap` without thinking whats cover behind.
If you're troll, you make it perfect, but if you're not I feel sorry for you.
![image](https://user-images.githubusercontent.com/23298059/232316428-f6704701-ddf9-4b5c-b0f5-e30210b0deaa.png)

**Comment by antonagestam** [comment]
<img width="314" alt="Skärmavbild 2023-04-16 kl  20 24 50" src="https://user-images.githubusercontent.com/218551/232333770-bb60a1a1-cd3b-4015-bdf9-ca30d276207b.png">

@streanger Huh, [what a bizarre situation](https://github.com/antonagestam/nmap/network/dependents). I pondered making installation print "INSTALLING MALWARE" with a really slow progress bar, just to teach people not to install unknown packages.

I've now removed the only release of this package from PyPI, and will archive this github project.
